workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -555,7 +555,7 @@ namespace <#= qualifiedNamespace #>
                         dirtyEvents += events<#= eventDetail.EventName #>.Count;
 <# } #>
 
-                        if (data.DirtyBit || dirtyEvents > 0)
+                        if (data.IsDirty() || dirtyEvents > 0)
```

Comment:
Drive-by. We should also change the name of dirty events.

Reply:
Fixed!